samples/security/spire/spire-quickstart.yaml

```diff
@@ -453,6 +453,7 @@ data:
           # Minikube does not have a cert in the cluster CA bundle that
           # can authenticate the kubelet cert, so skip validation.
           skip_kubelet_verification = true
+          disable_container_selectors = true
```

Comment:
nit: maybe we could add a explanatory comment for this config as well

Reply:
Add comment: `If true, container selectors are not produced. This can be used to produce pod selectors when the workload pod is known but the workload container is not ready at the time of attestation.`

Reply:
done